.scripts/CreateNuget.ps1

```diff
@@ -0,0 +1,201 @@
+[CmdletBinding()]
+param(
+        # Path to msbuild.exe
+        [Parameter(Mandatory=$true)]
+        $msbuild,
+        
+        # Path to nuget.exe
+        [Parameter(Mandatory=$true)]
+        $nuget,
+
+        # Directory in which nuget should be created
+        [Parameter(Mandatory=$true)]
+        $outputDir,
+
+        # Nuget.org Api Key
+        [Parameter(Mandatory=$true)]
+        $ApiKey
+)
+
+<#
+    .SYNOPSIS
+    Updates version number in nuspec
+#>
+function UpdateVersionInNuspec{
+    [CmdletBinding()] param([string] $nuspecPath, [string] $version)
+
+    $xml = New-Object XML
+    $xml.Load($nuspecPath)
+    
+    $element = $xml.SelectSingleNode("//package/metadata/version")
+    $element.InnerText = $version
+
+    $xml.Save($nuspecPath)
+}
+
+<#
+    .SYNOPSIS
+    Finds latest F* windows binary package available on Github Releases.
+#>
+function FindBinaryPackage{
+    [CmdletBinding()] param([string] $repo)
+    
+    $releasesUri = "https://api.github.com/repos/$repo/releases"
+    $releases = 
+        Invoke-WebRequest $releasesUri | 
+        ConvertFrom-Json 
+        
+    $releases = 
+        $releases | % {
+            [PSCustomObject]@{
+                name = $_.name
+                tag_name = $_.tag_name
+                # For some reason '\d{4}' or '[0-9]{4}' do not work
+                asset = @($_.assets | Where-Object { $_.name -like 'fstar_[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9][0-9]_Windows_x64*'} | % { [PSCustomObject]@{ name = $_.name; url = $_.browser_download_url }})
+            } 
+        } |
+        Where-Object { $_.asset.Count -eq 1 }
+
+    if ($releases.Count -eq 0) {
+        throw "Could not find a suitable windows binary package"
+    }
+
+    $package = 
+        [PSCustomObject]@{
+            name = $releases[0].name
+            tag_name = $releases[0].tag_name
+            version = "0.$($releases[0].asset.name.Substring(6, 10))" # The date part of the filename
```

Comment:
I've set nuget versions as: 0.YYYY.MM.DD. Ideally, the weekly (pre-release) packages would be separated from full-blown releases. Some options that I can see are:
- keep this as is now
- add some '-pre' suffix to the version so that nuget.org treats them as pre-release (you need an extra step in Visual Studio to see them with that)
- use the major version for full-blown releases and parameterize the script to produce X.YYYY.MM.DD, where X is the number that describes a major version
What do you think?